docs/content/en/docs/implementing/tasks/_index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Working with Keptn tasks
+description: Learn how to work with Keptn tasks
+weight: 90
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Keptn tasks are defined in a
+[KeptnTaskDefinition](../../yaml-crd-ref/taskdefinition.md/)
+resource.
+A task definition includes a function
+that defines the action taken by that task.
+It can be configured in one of three different ways:
+
+- inline
+- referring to an HTTP script
+- referring to another `KeptnTaskDefinition`
+- referring to a
+  [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/)
+  resource that is populated with the function to execute
+
+### Context
+
+A Kubernetes context is a set of access parameters
+that contains a Kubernetes cluster, a user, a namespace,
+the application name, workload name, and version.
+For more information, see
+[Configure Access to Multiple Clusters](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+
+You may need to include context information in the `function` code
+included in the YAML file that defines a
+[KeptnTaskDefinition](../../yaml-crd-ref/taskdefinition.md)
+resource.
+For an example of how to do this, see the
+[keptn-tasks.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/keptn-tasks.yaml)
+file.
+
+A context environment variable is available via `Deno.env.get("CONTEXT")`.
+It can be used like this:
+  
+```javascript
+let context = Deno.env.get("CONTEXT");
+    
+if (context.objectType == "Application") {
+    let application_name = contextdata.appName;
+    let application_version = contextdata.appVersion;
+}       
+        
+if (context.objectType == "Workload") {
+    let application_name = contextdata.appName;
+    let workload_name = contextdata.workloadName;
+    let workload_version = contextdata.workloadVersion;
+}
+```
+
+## Parameterized functions
+
+`KeptnTaskDefinition`s can use input parameters.
```

Comment:
simple parameters are passed as a single map of key values, while the secret parameters are stored as a single Kubernetes secret.

Reply:
I added your sentence -- definite improvement!